pallets/identity/rpc/runtime-api/src/lib.rs

```diff
@@ -4,6 +4,7 @@
 
 use codec::Codec;
 use frame_support::dispatch::Vec;
+#[allow(unused_imports)]
```

Comment:
Can we just get rid of the imports if they aren't needed?

Reply:
removed.